### ISSUE TITLE
labels do not match images in the cropping results (UNETER-BTCV tutorial)

### ISSUE THREAD

**Issue by JunMa11** [Project-MONAI/research-contributions#35]
**Describe the bug**

I visualize the image-label patch pairs in the following tutorial.
https://github.com/Project-MONAI/tutorials/blob/master/3d_segmentation/unetr_btcv_segmentation_3d.ipynb
It seems that the label does not match the images. Some examples are provided as follows

![1642279461(1)](https://user-images.githubusercontent.com/19947331/149637128-3de17930-c8fd-48cb-9639-cbe44486bc97.png)

![1642279499(1)](https://user-images.githubusercontent.com/19947331/149637144-3b5ab0cc-72b9-43e3-ad10-9c88a2cbc59c.png)

**To Reproduce**
1. Go to https://github.com/Project-MONAI/tutorials/blob/master/3d_segmentation/unetr_btcv_segmentation_3d.ipynb
2. After finishing the data loading, run commands 

```python
case_num = 0
crop_id = 1
img_name = os.path.split(train_ds[case_num][crop_id]["image_meta_dict"]["filename_or_obj"])[1]
label_name = os.path.split(train_ds[case_num][crop_id]["label_meta_dict"]["filename_or_obj"])[1]
img = train_ds[case_num][crop_id]["image"]
label = train_ds[case_num][crop_id]["label"]

img_shape = img.shape
label_shape = label.shape
print(f"image shape: {img_shape}, label shape: {label_shape}")
plt.figure("image", (18, 6))
plt.subplot(1, 2, 1)
plt.title(img_name)
plt.imshow(img[0, :, :, 36].detach().cpu(), cmap="gray")
plt.subplot(1, 2, 2)
plt.title(label_name)
plt.imshow(label[0, :, :, 36].detach().cpu())
plt.show()

```

**Comment by wyli** [comment]
Probably you run into this issue https://github.com/Project-MONAI/MONAI/discussions/2827

**Comment by JunMa11** [comment]
Hi @wyli ,

The problem has been fixed! Thanks for your help very much:) 

I started to learn MONAI during the past few months. It is extremely flexible and I enjoy using it very much. 
Thank you and your colleagues for this awesome work!

Kindest regards,
Jun
